packages/web-console/src/utils/questdb.ts

```diff
@@ -302,4 +313,14 @@ export class Client {
   async showColumns(table: string): Promise<QueryResult<Column>> {
     return await this.query<Column>(`SHOW COLUMNS FROM '${table}';`)
   }
+
+  async getLatestRelease() {
+    try {
+      const response: Response = await fetch(`https://api.github.com/repos/questdb/questdb/releases/latest`)
+      return (await response.json()) as Release
+    } catch (error) {
+      // eslint-disable-next-line prefer-promise-reject-errors
+      return await Promise.reject(error)
```

Comment:
```suggestion
      throw error
```

since `getLatestRelease` is marked with `async`, there shouldn't be a
need to return promise, can just `throw`